src/test/java/com/google/devtools/build/android/dexer/BUILD

```diff
@@ -47,7 +48,10 @@ java_library(
 
 java_library(
     name = "testdata",
-    srcs = glob(["testdata/**/*.java"]),
+    srcs = glob(
```

Comment:
Might need to run buildifier and google-java-format against the changes in this PR.

Reply:
The code will get formatted automatically on import I think, so I don't think we have to manually invoke the formatter